### PR TITLE
chore: replace test mode with simulated mode

### PR DESCRIPTION
**Pull request by thachdoSC**
replace references to test mode with simulated mode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ $ npm start
 ```
 `note` The server will automatically restart and load any code changes you do.
 
-Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [test mode](https://smartcar.com/docs/guides/testing/), so you can log in with any username and password and you will see information of a simulated vehicle.
+Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [simulated mode](https://smartcar.com/docs/getting-started/dashboard/simulator), so you can log in with any username and password and you will see information of a simulated vehicle.
```
Comment (bdr99) [inline README.md]: This says you can log in with any username and password, but isn't that no longer true for simulated mode?

Comment (thachdoSC) [inline README.md]:
```suggestion
Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [simulated mode](https://smartcar.com/docs/getting-started/dashboard/simulator), which will allow you log in with simulated credentials or any username and password.
```

File: README.md [hunk 2]
```diff
@@ -26,4 +26,4 @@ $ npm start
 ```
 `note` The server will automatically restart and load any code changes you do.
 
-Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [test mode](https://smartcar.com/docs/guides/testing/), so you can log in with any username and password and you will see information of a simulated vehicle.
+Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [simulated mode](https://smartcar.com/docs/getting-started/dashboard/simulator), which will allow you log in with simulated credentials or any username and password.
```
Comment (thachdoSC) [inline README.md]:
```suggestion
Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [simulated mode](https://smartcar.com/docs/getting-started/dashboard/simulator), which will allow you to log in with simulated credentials or any username and password.
```

File: README.md [hunk 3]
```diff
@@ -26,4 +26,4 @@ $ npm start
 ```
 `note` The server will automatically restart and load any code changes you do.
 
-Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [test mode](https://smartcar.com/docs/guides/testing/), so you can log in with any username and password and you will see information of a simulated vehicle.
+Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [simulated mode](https://smartcar.com/docs/getting-started/dashboard/simulator), which will allow you to log in with simulated credentials or any username and password.
```
Comment (thachdoSC) [inline README.md]:
```suggestion
Once your server is up and running, you can authenticate your vehicle at `http://localhost:8000/login` in your browser. In our current set up, we are using Smartcar's [simulated mode](https://smartcar.com/docs/getting-started/dashboard/simulator), so you can log in with simulated credentials found in your Smartcar Dashboard or with any username and password.
```

**Review by qwenballard — APPROVED** [review]

**Pull request merged**